js/main.js

```diff
@@ -1,330 +0,0 @@
-
-    }
-
-  }
-
-  // converts kelvin into celcius and farenheit
-// returns kelvin, celcius and farenheit
-function tempConvert(K) {
-    const C = K-273.15;
-    const F = ((K-273.15)*1.8)+32;
-    return [Math.trunc(K),Math.trunc(C),Math.trunc(F)]
-}
-
-function addDataDOM(zip) {
-    const toStr = cardNum.toString()
-    document.getElementById('cardDivx'+toStr).setAttribute('style','width: 18rem;')
-    const img = document.getElementById('cardImgx'+toStr)
-    const icon = Weather.data.weather[0].icon
-    const title = document.getElementById('cardTitlex'+toStr)
-    const text = document.getElementById('cardTextx'+toStr)
-    const tempk = document.getElementById('cardli1x'+toStr)
-    const tempc = document.getElementById('cardli2x'+toStr)
-    const tempf = document.getElementById('cardli3x'+toStr)
-    const temps = tempConvert(Weather.data.main.temp)
-    
-
-    title.innerHTML = Weather.data.name;
-    text.innerHTML = Weather.data.weather[0].main + ', ' + Weather.data.weather[0].description;
-    tempk.innerHTML = tempk.innerHTML + "  " + temps[0]
-    tempc.innerHTML = tempc.innerHTML + "  " + temps[1]
-    tempf.innerHTML = tempf.innerHTML + "  " + temps[2]
-
-    img.setAttribute('src',`http://openweathermap.org/img/wn/${icon}@2x.png`);
-    img.setAttribute('alt',Weather.data.weather[0].main)
-
-    const delBtn = document.getElementById('cardBtnDelx'+toStr);
-    const refBtn = document.getElementById('cardBtnRefx'+toStr);
-    delRefBtn(delBtn,refBtn,zip,toStr)
-
-
-
-    // Image display
-//`http://openweathermap.org/img/wn/${icon}@2x.png`
-}
-
-// click event listeners Update function and delete function
-
-    //delBtn.addEventListener('click',)
-    //refBtn.addEventListener('click',)
-function delRefBtn(del,ref,zip,toStr){
-    del.addEventListener('click',delCard)
-    ref.addEventListener('click',function(){ refCard(zip, toStr); })
-}
-function delCard() {
-    console.log('del')
-
-}
-function refCard(zip, toStr) {
-    console.log('ref')
-    console.log('card ID',zip ,'#',toStr)
-}
-
-
-
-
-//INIT()
-ObjElemCreate(headerDivElem)
-addContent(headerDivContent)
-
-createElem('div','cardsx'+cardNum.toString(),'bodyDiv','container-fluid')
-// cardRowx: ['div','cardRowx','cardsx','row justify-content-around 1'],
-createElem('div','cardRowx'+cardNum.toString(),'cardsx0','row justify-content-around 1')
-
-//Button in headerDivElem.cardButton needs type Attribute assigned
-document.getElementById('cardButton').setAttribute('type','button')
-
-
-
-
-
-document.getElementById('cardInput').setAttribute('type','number')
-document.getElementById('cardInput').setAttribute('placeholder','10001')
-document.getElementById('cardInput').setAttribute('min','10001')
-document.getElementById('cardInput').setAttribute('max', '99950')
-
-// document.getElementById('cardImgx').setAttribute('src','')
-// document.getElementById('cardImgx').setAttribute('alt','...')
-document.getElementById('cardButton').addEventListener('click', BtnHello)
```

Comment:
Again would like to see these in a function that is called with the stuff from up top.